app/sdil/models/ReturnsVariation.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sdil.models
+
+import play.api.libs.json.{Format, Json}
+import sdil.models.backend.{Site, UkAddress}
+
+case class ReturnsVariation(
+  orgName: String,
+  ppobAddress: UkAddress,
+  importer: (Boolean, (Long,Long)) = (false, (0,0)),
```

Comment:
what would (true,(0,0)) mean? or (false,(1,1))?

Reply:
true (0,0) would be an anomoly, but false (1,1) would not. 

I guess both packer and importer could be set to (0,0) if they were not new but I thought it was clearer in the code to explicitly say they are new importer/copacker and the vols are..

Reply:
ah I had assumed the boolean referred to if there was any activity, not any change.